src/app/types.ts

```diff
@@ -9,4 +9,12 @@ export type State = {
 export type Request = {
     state: 'PENDING' | 'ACTIVE' | 'LAGGING' | 'SUCCEEDED' | 'FAILED';
     error: string;
-};
\ No newline at end of file
+};
+
+export type ThemeName = 'light' | 'dark';
```

Comment:
I don't think you'll need this. Instead add a `name` to `Theme`, so that a theme-switcher-component can render the theme names